src/mol-canvas3d/passes/marking.ts

```diff
@@ -113,8 +115,10 @@ export class MarkingPass {
         const { values: overlayValues } = this.overlay;
         ValueCell.update(overlayValues.uHighlightEdgeColor, Color.toVec3Normalized(overlayValues.uHighlightEdgeColor.ref.value, highlightEdgeColor));
         ValueCell.update(overlayValues.uSelectEdgeColor, Color.toVec3Normalized(overlayValues.uSelectEdgeColor.ref.value, selectEdgeColor));
```

Comment:
Should these 2 get `updateIfChanged` as well? Perhaps add one more version of `updateIfChanged` which accepts a comparator function, e.g., ``ValueCell.updateIfChangedCmp(a, b, Vec3.equals)``?

Reply:
Could be used in other places with `ValueCell.updateIfChangedCmp(a, b, shallowEquals)` too.

Reply:
Sounds good. In the context of uniforms it is not that important as they get uploaded tot the GPU every time anyway.